### PR TITLE
feat: customer addresses repo implementation

### PR DESCRIPTION
**Pull request by rob-mohc**

### CODE REVIEW

**Review by Ianmohc — APPROVED** [review]

File: customer-address.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+		createURL,
+	}
+}
+
+func (r customerAddressRepository) Delete(id int64, addressID int64) error {
+	url := r.createURL(fmt.Sprintf("customers/%v/addresses/%v.json", id, addressID))
+
+	_, _, err := r.client.Delete(url, nil)
+	if err != nil {
+		return err
```
Comment (Ianmohc) [inline customer-address.go:26]: What will you do if the delete fails?

Comment (rob-mohc) [reply customer-address.go:26]: I guess returning the error only and then is handled accordingly in FEMS? I took as example the Delete in `fulfillment-events.go`. You think we should add something else?

Comment (Ianmohc) [reply customer-address.go:26]: Don't add anything unless you are using it. So if you are happy to have the error as it is and catch it then its fine.

Comment (rob-mohc) [reply customer-address.go:26]: I think so.. According to designs, the FE is only providing a generic error message to the user when something goes wrong with deleting an address. It doesn't give any details. The network tab probably would just indicate whatever we send from the FEMS 👍🏻

**Pull request merged**